.github/scripts/test_tekton_tasks.sh

```diff
@@ -22,6 +22,59 @@ shopt -s nullglob
 
 WORKSPACE_TEMPLATE=${BASH_SOURCE%/*/*}/resources/workspace-template.yaml
 
+if [ -z "${TRUSTED_ARTIFACT_OCI_STORAGE}" ]
+then
+  echo "Error: env variable TRUSTED_ARTIFACT_OCI_STORAGE not defined"
+  echo ""
+  echo "It must point to a location in quay where trusted artifacts will be placed."
```

Comment:
This doesn't have to be in quay, right? In fact, the update that you did to the github workflow uses something else (a locally deployed registry).

Reply:
comment was based on outdated code

Reply:
ok